### ISSUE TITLE
Ignore vendor directory in example .phpcs.xml.dist in README.md

### ISSUE THREAD

**Issue by salcode** [WebDevStudios/php-coding-standards#22]
I love that the example `.phpcs.xml.dist` in the `README.md` is so short, so I'm hesitant to suggest adding anything to it.

I imagine a full fledged example `.phpcs.xml.dist` in the wiki might be useful in the future.

Despite all of this, I think it is worth considering excluding the `vendor` directory in the example by adding

```
<!-- Exclude the Composer Vendor directory. -->
<exclude-pattern>/vendor/*</exclude-pattern>
```

so if someone does run something like `./vendor/bin/phpcs .` the process won't hang as all the files in the vendor directory are linted (and ultimately run out of memory).

I mention this because when I set this up in a test project, I did exactly this 🤷‍♂

**Comment by aubreypwd** [comment]
I agree! We could even do this in our own ruleset config vs modifying the example. I don't see ever wanting to lint any vendor/ or dist/ or node_modules/ ( I bet there's a bunch to consider )....
